tests/ssl-opt.sh

```diff
@@ -1585,6 +1585,19 @@ CLI_OUT="cli_out.$$"
 PXY_OUT="pxy_out.$$"
 SESSION="session.$$"
 
+# If MBEDTLS_ENTROPY_C && MBEDTLS_ENTROPY_NV_SEED && !MBEDTLS_NO_PLATFORM_ENTROPY
+# create seedfile to fix test fail.
+requires_config_enabled MBEDTLS_ENTROPY_C
+requires_config_enabled MBEDTLS_ENTROPY_NV_SEED
+requires_config_disabled MBEDTLS_NO_PLATFORM_ENTROPY
+create_seedfile()
+{
+    [ -f $1 ] || dd if=/dev/urandom of=$1 bs=64 count=1 2>/dev/null
+}
+if [ "${SKIP_NEXT:-}" != "YES" ]; then
+    eval "create_seedfile `$P_QUERY MBEDTLS_PLATFORM_STD_NV_SEED_FILE`"
```

Comment:
No need for `eval` here.

Reply:
That's for removing double quotes which return by `$P_QUERY`